WooCommerce/Classes/Authentication/Application Password/ApplicationPasswordAuthorizationWebViewController.swift

```diff
@@ -254,6 +281,13 @@ extension ApplicationPasswordAuthorizationWebViewController: WKNavigationDelegat
     }
 }
 
+extension ApplicationPasswordAuthorizationWebViewController: UIAdaptivePresentationControllerDelegate {
+    func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
+        // This will be called when the error UI is dismissed
+        navigationController?.dismiss(animated: true)
+    }
+}
+
```

Comment:
Is this still in use? I don't see where we present any UI.

Reply:
You're right, it seems unused, so I removed it 3f40c59fc1383b7881b655fb533c1c3aa81507dc